### ISSUE TITLE
Modals and tables displays decimal numbers differently 

### ISSUE THREAD

**Issue by mdyring** [e-money/lunie-light#35]
Notice how last price uses a "." for decimals and the modal uses "," - we should use the same format, ideally whatever the browser is configured to use.

![image](https://user-images.githubusercontent.com/18033781/101535317-c520b780-3998-11eb-86df-9da13df451bc.png)

Related to #30

**Comment by Bitcoinera** [comment]
Oops, good catch. Will check

**Comment by Bitcoinera** [comment]
?? I cannot reproduce

![image](https://user-images.githubusercontent.com/40721795/101651307-331eba80-3a3d-11eb-8d13-2c841c1fd291.png)


Are you sure it is not 6 thousand what is being displayed there? :eyes:  weird

**Comment by mdyring** [comment]
Yeah, it is for locales that use comma as decimal separator (see https://en.wikipedia.org/wiki/Decimal_separator for discussion).

I believe this can be fixed by using Intl.NumberFormat consistency and abandoning toFixed

On 9 Dec 2020, at 16.40, Ana G. <notifications@github.com <mailto:notifications@github.com> > wrote:


?? I cannot reproduce


Are you sure it is not 6 thousand what is being displayed there? 👀 weird
